tests/src/system/basic/WskBasicNodeTests.scala

```diff
@@ -147,181 +145,6 @@ class WskBasicNodeTests
-    }
-
-    // TODO: remove this tests and its assets when "whisk.js" is removed entirely as it is no longer necessary
-    it should "Ensure that whisk.invoke() still uses a callback when provided one" in withAssetCleaner(wskprops) {
-        val expectedDuration = 3.seconds
-
-        (wp, assetHelper) =>
-            val asyncName = "ThreeSecondRule"
-            val asyncAction = Some(TestUtils.getTestActionFilename("threeSecondRule.js"))
-
-            assetHelper.withCleaner(wsk.action, asyncName) {
-                (action, _) =>
-                    action.create(asyncName, asyncAction)
-            }
-
-            // this action supplies a 'next' callback to whisk.invoke()
-            val invokeActionName = "invokeAction"
-            val invokeAction = Some(TestUtils.getTestActionFilename("invokeCallback.js"))
-
-            assetHelper.withCleaner(wsk.action, invokeActionName) {
-                (action, _) =>
-                    action.create(invokeActionName, invokeAction)
-            }
-
-            var start = System.currentTimeMillis()
-            val runResolve = wsk.action.invoke(invokeActionName, Map("resolveOrReject" -> "resolve".toJson))
-            withActivation(wsk.activation, runResolve) {
-                activation =>
-                    val result = activation.response.result.get
-                    result.fields.get("activationId") shouldBe defined
-                    result.fields.get("error") should not be defined
-                    result.getFieldPath("result", "message") should be(Some {
-                        "Three second rule!".toJson
-                    })
-
-                    val duration = System.currentTimeMillis() - start
-                    duration should be >= expectedDuration.toMillis
-            }
-
-            start = System.currentTimeMillis()
-            val runReject = wsk.action.invoke(invokeActionName, Map("resolveOrReject" -> "reject".toJson))
-            withActivation(wsk.activation, runReject) {
-                activation =>
-                    val result = activation.response.result.get
-                    result.fields.get("activationId") should not be defined
-                    result.getFieldPath("error", "message") should be(Some {
-                        "Three second rule!".toJson
-                    })
-
-                    val duration = System.currentTimeMillis() - start
-                    duration should be >= expectedDuration.toMillis
-            }
-    }
-
-    // TODO: remove this tests and its assets when "whisk.js" is removed entirely as it is no longer necessary
-    it should "Ensure that whisk.trigger() still uses a callback when provided one" in withAssetCleaner(wskprops) {
-        (wp, assetHelper) =>
-            // this action supplies a 'next' callback to whisk.trigger()
-            val nameOfActionThatTriggers = "triggerAction"
-            val actionThatTriggers = Some(TestUtils.getTestActionFilename("triggerCallback.js"))
```

Comment:
@jasonpet Looks like we can also remove `triggerCallback.js` and `triggerPromise.js`